app/services/foreman_ipxe/ipxe_message_renderer.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ForemanIpxe
+  class IpxeMessageRenderer
+    attr_accessor :message
+
+    def initialize(message:)
```

Comment:
Could you give this an explicit default value - `message: nil` or just anything really, that colon tickles some kind of nerve on me.